src/client/theme-default/components/VPDoc.vue

```diff
@@ -11,6 +12,8 @@ const { hasSidebar } = useSidebar()
 const pageName = computed(() => {
   return page.value.relativePath.slice(0, page.value.relativePath.indexOf('/'))
 })
+
+useCopyCode()
```

Comment:
Maybe we should use this in `VPContent` instead? Because people can use `layout: page` and still have code block 👀

Reply:
@kiaking All of our styles regarding code blocks need parent to have `vp-doc` class (all references to `[class*='language-']` are present only inside `vp-doc.css` file). When `layout: page` is set, we only have syntax highlighting, no other thing like language descriptor at the top or that curved-rectangular container is there.

But since `useCopyCode` just adds the event handlers not the styles, there isn't any issue in adding it to `VPContent` itself. Developers will need to add content to `span.copy` in order to interact with the element and copy in page layouts.

Reply:
Ahhh good point. I did miss out one that part 😅 

But thanks! I think your call of moving the code to `VPContent` still make sense, where the purpose of `layout: page` is to provide same markdown pasring feature, but without styles 👍

As you mentioned, users can define their own styles for this button inside page layout.